benchmarks/experiment_runner.py

```diff
@@ -879,6 +880,12 @@ def __str__(self):
       default=False,
       help="""If set, verifies the model output with PT Eager mode, and saves relative error to the output file."""
   )
+  parser.add_argument(
+      "--strict-compatible",
+      action="store_true",
+      default=False,
```

Comment:
I believe that setting `action="store_true"` already sets the default accordingly.

Reply:
Hmm, right. I also removed other default=False when action="store_true" in the file.